src/sdk/http/http-client.ts

```diff
@@ -99,12 +99,22 @@ export abstract class HttpClient<TRequest, TResponse> implements IHttpClient {
 		}
 	}
 
+	static getFilteredParameters(params?: any): any {
+		let filteredParams = {};
+		for (const param in params) {
+			if (params[param] !== undefined) {
```

Comment:
Ensure `filteredParams` is defined here. `if (!filteredParams) { filteredParams = {}; }`

Reply:
will do